src/constants.js

```diff
@@ -2,4 +2,3 @@ export const DOMAIN = 'accessible-colors.com';
 export const SITE_URL = `http://${DOMAIN}`;
 export const REPO = 'moroshko/accessible-colors';
 export const MIN_FONT_SIZE_IN_PX = 8;
-export const MAX_FONT_SIZE_IN_PX = 36;
```

Comment:
The max size we choose will be somewhat arbitrary. Once I make the other changes you commented on above, I will choose the largest number that looks good in the preview (when input is in pt).